src/main/java/cn/nukkit/block/BlockFurnaceBurning.java

```diff
@@ -128,4 +129,14 @@ public boolean canHarvestWithHand() {
     public BlockFace getBlockFace() {
         return BlockFace.fromHorizontalIndex(this.getMeta() & 0x7);
     }
+
+    protected BlockEntityType<? extends Furnace> getEntityType() {
```

Comment:
Should use a factory method for this so the `BlockEntityType` can be passed through the constructor

Reply:
Created a factory for this.